LCSC-API.md

```diff
@@ -1,10 +1,11 @@
 # LCSC API
 
-- to get a product page based on LCSC number use GET request on
+1. To get a product page based on LCSC number, use GET request on
```

Comment:
What's the reason for making this numeric list? There isn't a clear sequence "first this and then this". The bullets serve as a clear graphical division between two parts of the text.

Reply:
Numerical lists allow for discussion among teams better, which is important for software like this which may be prone to forking/external use. Either a short subheading or ordered list is most appropriate here, in my opinion.

Am happy to revert the list type change, if requested.